.github/ISSUE_TEMPLATE/new-event.md

```diff
@@ -0,0 +1,62 @@
+---
+name: New Event
+about: Submitting new events to the calendar
+title: "[EVENT]"
+labels: New event
+assignees: ''
+
+---
+
+## Event Name
+
+## Date
+
+* DATE HERE
```

Comment:
Date format would be useful

Reply:
Went for `YEAR-MONTH-DAY` because I love ISO dates, though probably a nicer way to write it out.

Reply:
I mean, ISO-8601 is objectively the best date format 😎